### PR TITLE
TRACING-895 Documenting autoscaling the Collector and Ingester.

### PR DESCRIPTION
**Pull request by JStickler**
Take two (on the correct branch this time).  Documenting the Jaeger autoscaling and some minor cleanup in the tables.

### CODE REVIEW

**Comment by JStickler** [comment]
Updated topics are in the preview under Jaeger > Jaeger Installation > Deploying Jaeger

File: modules/jaeger-config-ingester.adoc [hunk 1]
```diff
@@ -88,3 +81,28 @@ spec:
...
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-streaming
+spec:
+  strategy: streaming
+  ingester:
+    maxReplicas: 8
```
Comment (jpkrohling) [inline modules/jaeger-config-ingester.adoc:103]: Just out of curiosity: why 8 for the ingester, but 5 for the collector? Is this also so in the upstream docs?

**Review by jpkrohling — APPROVED** [review]

File: modules/jaeger-config-ingester.adoc [hunk 2]
```diff
@@ -6,57 +6,50 @@ This REFERENCE module included in the following assemblies:
...
 |
-|
 
-|ingester:
- options:
-  kafka:
-    consumer:
-      topic: jaeger-spans
+|autoscale:
+|This parameter controls whether to enable/disable autoscaling for the Ingester.  Set to `false` to explicitly disable autoscaling.
```
Comment (kevinearls) [inline modules/jaeger-config-ingester.adoc:24]: Just a nitpick here, but it might be helpful to word this so it's clear autoscale is true by default.

**Review by kevinearls — APPROVED** [review]
LGTM

File: modules/jaeger-config-ingester.adoc (hunk 1 above)
Comment (JStickler) [reply modules/jaeger-config-ingester.adoc:103]: That's from the upstream docs.

File: modules/jaeger-config-ingester.adoc [hunk 3]
```diff
@@ -85,3 +80,28 @@ spec:
       es:
         server-urls: http://elasticsearch:9200
 ----
+
+== Configuring Ingester for autoscaling
+
+The Ingester can be configured to autoscale on demand. You configure autoscaling by setting the `autoscale` parameter to `true` and specifying a value for `.spec.ingester.maxReplicas` along with a reasonable value for the resources that you expect the Ingester's pod to consume. If you do not set a value for `.spec.ingester.maxReplicas` the Operator will set it to `100`.
```
Comment (neal-timpe) [inline modules/jaeger-config-ingester.adoc:86]: I think the last part of the last sentence 

>the Operator will set it to `100`. 

Should be:

>the default is `100`.

I think the operator less anthropomorphic.

File: modules/jaeger-config-collector.adoc [hunk 4]
```diff
@@ -36,4 +40,46 @@ The collectors are stateless and thus many instances of Jaeger Collector can be
...
+|Integer, for example, `2000`.
+
+|replicas:
+|Specifies the number of Collector replicas to create.
+|Integer, for example, `5`.
 |===
+
+
+== Configuring the Collector for autoscaling
+
```
Comment (neal-timpe) [inline modules/jaeger-config-collector.adoc:63]: I think both the Collector and Ingester sections could benefit from a sentence that tells me What does configuring the Collector and Ingester to autoscale on demand do for me? Why would I want to do that?

**Review by neal-timpe — APPROVED** [review]

**Pull request merged**